model/event/subscribers.go

```diff
@@ -197,6 +198,13 @@ func NewEnqueuePatchSubscriber() Subscriber {
 	}
 }
 
+func NewRunChildPatchSubscriber() Subscriber {
+	return Subscriber{
+		Type:   RunChildPatchSubscriberType,
+		Target: nil,
```

Comment:
the target could contain information about what you want to do when the event gets triggered. 
If the subscriber contains the parent status we want to trigger on this would allow us to consolidate the three subscriptions into one on patch outcome and case on the subscriber.